packages/fxa-auth-server/lib/senders/oauth_client_info.js

```diff
@@ -28,8 +31,15 @@ module.exports = (log, config) => {
   async function fetch(clientId) {
     log.trace('fetch.start');
 
-    if (!clientId || clientId === 'sync') {
-      log.trace('fetch.sync');
+    // if there is no clientId, default to the more general 'Mozilla'
+    if (!clientId) {
+      log.trace('fetch.noClientId');
+      return MOZILLA_CLIENT;
```

Comment:
For the sync flow, if there is no service specified, that should default to Firefox. Is that OK here for `fx_desktop_v3`?

Reply:
Good to know that no service specified means sync - I'll revert some of the changes here.